client/app/scripts/liveblog-security.service.js

```diff
@@ -32,7 +32,7 @@ angular.module('liveblog.security', [])
             return $rootScope.currentUser._id === archive.original_creator || canPublishAPost();
         }
         function canAccessSettings(archive) {
-            return canCreateABlog() && isUserOwnerOrAdmin(archive);
+            return canCreateABlog() && (isUserOwnerOrAdmin(archive) || isMemberOfBlog(archive));
```

Comment:
You're right! Despite of what is understandable from the code, if you log in with contributor account you don't have the rights to create blogs. Tested on the branch.
So :+1: